Chap_API_Event.tex

```diff
@@ -565,10 +570,16 @@ \subsubsection{Completion Callback Function Status Codes}
 %
 \declareconstitemvalue{PMIX_EVENT_ACTION_DEFERRED}{-333}
 Event handler: Action deferred.
+\end{constantdesc}
 %
-\declareconstitemvalue{PMIX_EVENT_ACTION_COMPLETE}{-334}
-Event handler: Action complete.
+
+The following status code may be returned by a handler to stop the handler chain.  It will not appear in the \refarg{results} for any handler invocation since it is the last handler executed in the chain.
```

Comment:
```suggestion
The following status code may be returned by a handler to stop the handler chain.  It will not appear in the \refarg{results} for any handler invocation since the handler that produces it is the last handler executed in the chain.
```

Reply:
Ty and commited